lm_eval/tasks/hellaswag/hellaswag.yaml

```diff
@@ -0,0 +1,19 @@
+group:
+  - multiple_choice
+task: hellaswag
+dataset_path: hellaswag
+dataset_name: null
+output_type: multiple_choice
+training_split: train
+validation_split: validation
+test_split: null
+template_aliases: "{% set gold = label %}{% set answer_choices = endings|map('regex_replace', '\\[.*?\\]', '')|map('replace', ' [title]', '. ')|map('replace', '  ', ' ')|list %}" 
+doc_to_text: "{% set text = activity_label ~ ': ' ~ ctx_a ~ ' ' ~ ctx_b.capitalize() %}{{text|regex_replace('\\[.*?\\]', '')|replace(' [title]', '. ')|replace('  ', ' ')}}"
+doc_to_target: "{{gold}}"
```

Comment:
Note to self that after merging this will need to add gold_alias to #599 for this task before merging #599 .